### PR TITLE
JDF-655: Rename helloworld-bpms to helloworld-bpmsuite to be consistent ...

### PR DESCRIPTION
**Pull request by sgilda**
...with product documentation

### CODE REVIEW

File: helloworld-bpmsuite/README.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
-helloworld-bpms: A Basic BPMS example
+helloworld-bpmsuite: A Basic BPM Suite example
 ======================================
 Author: Rafael Benevides  
 Level: Beginner  
 Technologies: Drools, BRMS 
-Summary: Shows how to run a Process that was defined on a BPMS server  
-Target Product: BPMS  
+Summary: Shows how to run a Process that was defined on a JBoss BPM Suite server  
+Target Product: BPM-SUITE
```
Comment (rafabene) [inline helloworld-bpmsuite/README.md:7]: I'll need to include this on QSTools config. But it's fine to merge that way

**Comment by rafabene** [comment]
It's perfect! Thank you so much

**Comment by sgilda** [comment]
I can modify the QS tools for you! Thanks!

**Comment by sgilda** [comment]
@rafabene : Now that I think about it, shouldn't the `Target Product` be "BRMS" since it's in the BRMS quickstart repository and it really demonstrates using BRMS with BPM Suite?

**Pull request merged**